docs/documentation/platform/scim/overview.mdx

```diff
@@ -24,9 +28,14 @@ SCIM providers:
 **FAQ**
 
 <AccordionGroup>
-<Accordion title="Why do SCIM-provisioned users have to finish setting up their account?">
-    Infisical's SCIM implementation accounts for retaining the end-to-end encrypted architecture of Infisical because we decouple the **authentication** and **decryption** steps in the platform. 
-    
-    For this reason, SCIM-provisioned users are initialized but must finish setting up their account when logging in the first time by creating a master encryption/decryption key. With this implementation, IdPs and SCIM providers cannot and will not have access to the decryption key needed to decrypt your secrets.
-</Accordion>
-</AccordionGroup>
\ No newline at end of file
+  <Accordion title="Why do SCIM-provisioned users have to finish setting up their account?">
+    Infisical's SCIM implementation accounts for retaining the end-to-end
+    encrypted architecture of Infisical because we decouple the
+    **authentication** and **decryption** steps in the platform. For this
```

Comment:
auth and decryption is no longer decoupled, this is old info. sso works without user having to enter password any more